Makefile

```diff
@@ -0,0 +1,41 @@
+build:
```

Comment:
Why don't you just use https://github.com/mattermost/mattermost-plugin-sample/blob/master/Makefile?